### ISSUE TITLE
Slack is being shown as Asana in the dialog box

### ISSUE THREAD

**Issue by mattmothersbaugh** [robjschroeder/App-Auto-Patch#8]
Slack was properly updated it's just that the dialog box is listing Asana.

![Screenshot 2023-06-29 at 2 06 36 PM](https://github.com/robjschroeder/App-Auto-Patch/assets/104868992/8b9db6d9-b6f5-4f8c-8842-9e0eb40299c5)

```2023-06-29 14:03:25 - App Auto-Patch: Installing slack...
2023-06-29 14:03:25 - App Auto-Patch: [INFO] Update swiftDialog: progress: increment 5
2023-06-29 14:03:25 - App Auto-Patch: [INFO] Update swiftDialog: icon: /Applications/Asana.app
2023-06-29 14:03:25 - App Auto-Patch: [INFO] Update swiftDialog: progresstext: Checking Asana …
2023-06-29 14:03:25 - App Auto-Patch: [INFO] Update swiftDialog: listitem: index: 16, status: wait, statustext: Checking …
2023-06-29 14:03:25 : REQ   : slack : ################## Start Installomator v. 10.4, date 2023-06-02
2023-06-29 14:03:25 : INFO  : slack : ################## Version: 10.4
2023-06-29 14:03:25 : INFO  : slack : ################## Date: 2023-06-02
2023-06-29 14:03:25 : INFO  : slack : ################## slack
2023-06-29 14:03:26 : INFO  : slack : setting variable from argument BLOCKING_PROCESS_ACTION=prompt_user NOTIFY=silent LOGO=appstore IGNORE_APP_STORE_APPS=yes INTERRUPT_DND=no IGNORE_DND_APPS=firefox,Google Chrome,Safari,Microsoft Edge,Opera,Amphetamine,caffeinate,Caffeine
(eval):1: command not found: Chrome,Safari,Microsoft
2023-06-29 14:03:26 : INFO  : slack : setting variable from argument DIALOG_CMD_FILE="/var/tmp/dialog.patchomator.U3bFj"
2023-06-29 14:03:26 : INFO  : slack : setting variable from argument DIALOG_LIST_ITEM_NAME='Asana'
2023-06-29 14:03:26 : INFO  : slack : BLOCKING_PROCESS_ACTION=tell_user
2023-06-29 14:03:26 : INFO  : slack : NOTIFY=success
2023-06-29 14:03:26 : INFO  : slack : LOGGING=INFO
2023-06-29 14:03:26 : INFO  : slack : LOGO=/System/Applications/App Store.app/Contents/Resources/AppIcon.icns
2023-06-29 14:03:26 : INFO  : slack : Label type: dmg
2023-06-29 14:03:26 : INFO  : slack : archiveName: Slack.dmg
2023-06-29 14:03:26 : INFO  : slack : no blocking processes defined, using Slack as default
2023-06-29 14:03:26 : INFO  : slack : App(s) found: /Applications/Slack.app```

**Comment by memile123** [comment]
Same here. Seeing this in my environment as well. I thought it was something to do with the list item/statustext not updating. Overall, great tool and thank you for the work on this @robjschroeder

**Comment by AndrewMBarnett** [comment]
Any thoughts on the Slack item showing Asana?

**Comment by robjschroeder** [comment]
Fixed in merge #30 , thanks again @TechTrekkie
